src/components/aboutUs/AboutUs.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card } from "react-bootstrap"
```

Comment:
please use the card from the chakra library, instead of the default react card

Reply:
i copied the styling from CreateResources, you imported card from bootstrap unless i am mistaken

Reply:
sorry my previous comment didnt go through, ignore what I said as the version of chakra we're using doesn't contain the card component